src/components/pages/WikiNotePage.tsx

```diff
@@ -77,11 +78,17 @@ const WikiNotePage = ({ storage }: WikiNotePageProps) => {
 
   const { showSearchModal } = useSearchModal()
 
+  const storageTags = useMemo(() => {
+    if (storage == null) return []
+    return values(storage.tagMap).map((tag) => tag)
```

Comment:
`[...storage.tagMap.values()]`

Reply:
There is no values() on ObjectMap<T>

So final should be:
`return [...values(storage.tagMap)]`?
or maybe even
`return values(storage.tagMap)`?